### PR TITLE
[Update]: Making Homepage - cart responsive.

### PR DESCRIPTION
**Pull request by munjalSaksham71**
Make Homepage and Cartpage and Wishlist responsive.

### CODE REVIEW

File: pages/cart.css [hunk 1]
```diff
@@ -39,4 +39,12 @@ select {
 .btn_checkout {
     width: 19rem;
     height: 10rem;
+}
+
+@media screen and (max-width: 1200px) {
+    .cart_total_box{
+        position: relative;
+        top: 6rem;
+        left: -14rem;
```
Comment (dev-enforced) [inline pages/cart.css:48]: Instead of using left property you can even use right property to avoid giving large values to them.

**Review by dev-enforced — APPROVED** [review]
The code looks good to merge!! Good work on the responsiveness of the mentioned pages.Had a minor suggestion so you can check it out later.

File: pages/cart.css (hunk 1 above)
Comment (munjalSaksham71) [reply pages/cart.css:48]: Ok thanks- I will care for this in future.

**Review by mariyasada — APPROVED** [review]
looks good

**Pull request merged**